code/controllers/subsystem/ticker.dm

```diff
@@ -41,6 +41,9 @@ SUBSYSTEM_DEF(ticker)
 
 /datum/controller/subsystem/ticker/Initialize(timeofday)
 	load_mode()
+	
+	GLOB.respawntime = CONFIG_GET(number/marine_respawn)
+	GLOB.xenorespawntime = CONFIG_GET(number/xeno_respawn)
```

Comment:
i think its better to replace all GLOB.respawntime and GLOB.xenorespawntime for the CONFIG_GET

Reply:
No this is fine

Reply:
Oh ye thats right